core/base/inc/TString.h

```diff
@@ -166,6 +172,9 @@ operator+(T f, const TString &s);
 
 friend Bool_t  operator==(const TString &s1, const TString &s2);
 friend Bool_t  operator==(const TString &s1, const char *s2);
+#ifdef __cpp_lib_three_way_comparison
+friend std::strong_ordering operator<=>(const TString &s1, const TString &s2);
+#endif
```

Comment:
Is this needed, @bernhardmgruber @jalopezg-git ? It seems we're declaring a hidden friend and an "unhidden" friend at the same time?